corehq/blobs/interface.py

```diff
@@ -74,3 +82,19 @@ def get_identifier(basename):
         else:
             prefix = "unsafe"
         return prefix + "." + uuid4().hex
+
+    @staticmethod
+    def get_args_for_delete(identifier=NOT_SET, bucket=NOT_SET):
+        if identifier is NOT_SET and bucket is NOT_SET:
+            raise ArgumentError("'identifier' and/or 'bucket' is required")
+        if identifier is None:
+            raise ArgumentError("refusing to delete entire bucket when "
+                                "null blob identifier is provided (either "
+                                "provide a real blob identifier or pass "
+                                "bucket as keyword argument)")
```

Comment:
What would happen if `identifier` were `""` or `False` or something unexpected like that, do you know if that would fail somewhere further down the line?  I'm wondering if `identifier is None` is strict enough.